test/_run.js

```diff
@@ -390,4 +426,25 @@ module.exports = Emittery => {
 		const emitter = new Emittery();
 		t.throws(() => emitter.listenerCount(42), TypeError);
 	});
+
+	function catchUnhandledRejection(limit) {
+		return new PCancelable((onCancel, resolve) => {
```

Comment:
`PCancelable` doesn't seem necessary.

Reply:
I would have like to be able to cancel the `unhandledRejection` listener in case of failure, but it won't work with a cancellable promise. I will try something else.

Reply:
You can't observe test failures though.